serverlessworkflow/modules/ROOT/pages/data-index/data-index-usecase-multi.adoc

```diff
@@ -0,0 +1,221 @@
+= Deploying Data Index and multiple {product_name} application on Minikube
+:compat-mode!:
+// Metadata:
+:description:  Deploying Multiple {product_name} pushing to single Data Index on Minikube
+:keywords: kogito, workflow, quarkus, serverless, kubectl, minikube, operator, dataindex
+:table-caption: Data Set
+// envs for common content
+:kubectl_prereq: command-line tool is installed. Otherwise, Minikube handles it.
+//Common constants
+:data_index_ref: Data Index
+
+
+This document describes how to deploy a multiple {product_name} workflow applications and the {data_index_ref} service using a local Kubernetes cluster, such as link:{minikube_url}[Minikube], using the link:{kogito_serverless_operator_url}[{operator_name}].
+
+For more information about Minikube and related system requirements, see link:{minikube_url}/docs/start/[Getting started with Minikube] documentation.
+
+This use case is intended to represent an installation with:
+
+* A singleton Data Index Service with PostgreSQL persistence
+* The `greeting` workflow (no persistence), that is configured to register events to the Data Index Service.
+* The `helloworld` workflow (no persistence), that is configured to register events to the Data Index Service.
+* Both workflows are configured to register the process events on the {data_index_ref} Service.
+
+You can directly access the UseCase2 example application we are going to follow at link:{flow_examples_operator_url}[{product_name} Use Cases repository].
+
+// shared pre req
+include::common/_prerequisites.adoc[]
+
+You can check the Minikube installation by entering the following commands in a command terminal:
+
+.Verify Minikube version
+[source,shell]
+----
+minikube version
+----
+
+.Verify `kubectl` CLI version
+[source,shell]
+----
+kubectl version
+----
+
+[NOTE]
+====
+If `kubectl` is not installed, then Minikube handles it when you execute the following command:
+
+.`kubectl` is available using Minikube
+[source,shell]
+----
+alias kubectl="minikube kubectl --"
+----
+====
+
+.Procedure
+. After cloning the link:{flow_examples_operator_url}[SonataFlow Use Cases repository]. Open a terminal and run the following commands
+. Create the namespace:
```

Comment:
We can add the files to the guide inline directly instead. @kaldesai can we have an official OpenShift Serverless Logic examples/code repository?

Reply:
For this requirement this is too late, let's do a kie-issue or requirement to tackle this.

Reply:
:+1: for adding the directly here.